Android/app/src/go/intra/split/retrier.go

```diff
@@ -351,9 +351,11 @@ func copyOnce(dst io.Writer, src io.Reader) (int64, error) {
 
 func (r *retrier) ReadFrom(reader io.Reader) (bytes int64, err error) {
 	for !r.retryCompleted() {
-		if bytes, err = copyOnce(r, reader); err != nil {
+		var b int64
+		if b, err = copyOnce(r, reader); err != nil {
 			return
 		}
+		bytes += b
```

Comment:
`b` might be `> 0` even when `err != nil`, so we should always add it to `bytes` regardless of the `err`.

```suggestion
		n, e := copyOnce(r, reader)
		bytes += n
		if err = e; err != nil {
			return
		}
```

Reply:
Done. Thanks.